iBudget/utils/spendings_limit_checker.py

```diff
@@ -46,3 +47,51 @@ def comp_gr_spends_w_limit(group_id):
 
 
     return None
+
+def compare_ind_spend_limit(user, current_date, spending, current_value):
+    """Creating data for spending limitation tracking.
+        Args:
+            user (UserProfile): user.
+            current_date (date): date for limitation tracking
+            spending (SpendingCategories): spending category for tracking
+        Returns:
+            ... about current spending limitation on this category.
+    """
+    find_limit = \
+    SpendingLimitationIndividual.objects.filter(
+        Q(start_date__lte=current_date) &
+        Q(finish_date__gte=current_date)).filter(user=user, spending_category=spending)
+    if not find_limit:
+        return 'There are no individual limits on including this date period.'
+
+    response = ''
+    for item in find_limit:
+        total = sum(SpendingHistory.objects.filter(
+            owner=user,
+            spending_categories=spending,
+            date__range=[item.start_date, item.finish_date]).values_list('value', flat=True))
+        if total >= item.value:
+            response += "Warning! Your spending limit {} " \
```

Comment:
Why don't you use fancy `f""` formatting as in other code that you have added to this pull request?

Reply:
Ok. I'll do it